Mathlib/CategoryTheory/Limits/Shapes/Pullback/Pasting.lean

```diff
@@ -46,11 +46,13 @@ variable (h₁ : i₁ ≫ g₁ = f₁ ≫ i₂) (h₂ : i₂ ≫ g₂ = f₂ ≫
 
 /-- Given
 
+```
 X₁ - f₁ -> X₂ - f₂ -> X₃
```

Comment:
Perhaps worth doing the horizontal arrows too

Reply:
Note that the `f`s are inside the arrow, so I'm not sure whether that'd be more or less readable